integration/k8s/demo/org3/org3-peer0-svc.yaml

```diff
@@ -0,0 +1,36 @@
+# Copyright IBM Corp. All Rights Reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+---
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: peer0-org3
+  name: peer0-org3
+  namespace: hyperledger
+spec:
+  type: ClusterIP
+  ports:
+    - name: "peer-core"
+      port: 7051
+      targetPort: 7051
+  selector:
+    app: peer0-org3
+---
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: peer0-org3
+    metrics-service: "true"
```

Comment:
Is there a specific reason for setting up the metrics service here?
It does not seem necessary for this k8s deployment.

Reply:
true, let's remove it